### PR TITLE
Blog:Tauri Community Growth and Feedback

### PR DESCRIPTION
**Pull request by lorenzolewis**

### CODE REVIEW

File: blog/2023-02-09-tauri-community-growth-and-feedback.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+authors: [lorenzolewis, amrbashir]
+description: New v2.0 alpha released and new backend for Linux port
```
Comment (amrbashir) [inline blog/2023-02-09-tauri-community-growth-and-feedback.mdx]: copy pasta

**Review by amrbashir — CHANGES_REQUESTED** [review]
#

File: blog/2023-02-09-tauri-community-growth-and-feedback.mdx (hunk 1 above)
Comment (lorenzolewis) [reply blog/2023-02-09-tauri-community-growth-and-feedback.mdx]: Just fixed it, thanks for the catch!

**Review by FabianLars — APPROVED** [review]
ngl, i love your writing style

**Comment by lorenzolewis** [comment]
> ngl, i love your writing style

Aww thanks, that means a lot! <3

**Pull request merged**